error-handling/src/main/java/com/palantir/remoting/http/errors/SerializableError.java

```diff
@@ -35,13 +37,28 @@
 
     public abstract String getMessage();
 
-    public abstract Class<? extends Exception> getExceptionClass();
+    @JsonProperty("exceptionClass")
+    public abstract String getExceptionClassName();
```

Comment:
I'm not 100% sold on this. The current proposal explicitly avoids breaking any existing APIs so that (for example), people can freely mix and match different versions of http-remoting in their app. If my server relies on old behavior and I pull in a client that relies on new behavior, I don't want my server to break at runtime due to link errors.

On the other hand, the API break gets people using the "right" APIs, so... if you think it's worth it I can make the change.